container/container_unix.go

```diff
@@ -278,6 +295,29 @@ func (container *Container) UnmountSecrets() error {
 	return detachMounted(container.SecretMountPath())
 }
 
+// ConfigMounts returns the mounts for configs.
+func (container *Container) ConfigMounts() []Mount {
+	var mounts []Mount
+	for _, configRef := range container.ConfigReferences {
+		if configRef.File == nil {
+			continue
+		}
+		configPath, err := container.ConfigFilePath(configRef.File)
```

Comment:
I think we should encourage people to provide absolute paths, since the goal of this feature is to mount files in places where your application expects them. If the user doesn't provide an absolute path, putting the file in `/` is the most sane thing I can think of to do.